poonam/JS/Assignment6_js/string.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Document</title>
+        <script>
+            //trim
+            function trim(){
+                var str = document.getElementById("val").value;
+                var output = str.trim();
+                output = document.getElementById("res").innerHTML= "Output = "+output;
+            }
+            //length
+            function strlength(){
+                var str1 = document.getElementById("val1").value;
+                var output1 = str1.length;
+                output1 = document.getElementById("res1").innerHTML= "Output = "+output1;
+            }
+            //index
+            function index(){
+                var str2 = document.getElementById("val2").value;
+                var ind1 = document.getElementById("ind1").value;
+                var output2 = str2.indexOf(ind1);
+                output2 = document.getElementById("res2").innerHTML= "Output = "+output2;
+            }
+            //last index
+            function lastindex(){
+                var str3 = document.getElementById("val3").value;
+                var ind2 = document.getElementById("ind2").value;
+                var output3 = str3.lastIndexOf(ind2);
+                output3 = document.getElementById("res3").innerHTML= "Output = "+output3;
+            }
+            //concat
+            function concatstr(){
+                var cstr1 = document.getElementById("val4").value;
+                var cstr2 = document.getElementById("val5").value;
+                var output4 = cstr1.concat(cstr2);
+                output4 = document.getElementById("res4").innerHTML= "Output = "+output4;
+            }
+            //slice
+            function slicestr(){
+                var str5 = document.getElementById("val6").value;
+                var ind1 = document.getElementById("find").value;
+                var ind2 = document.getElementById("lind").value;
+                var output5 = str5.slice(ind1, ind2);
+                output5 = document.getElementById("res5").innerHTML= "Output = "+output5;
+            }
+            //lower case
+            function lower_case(){
+                var str6 = document.getElementById("val7").value;
+                var output6 = str6.toLowerCase();
+                output6 = document.getElementById("res6").innerHTML= "Output = "+output6;
+            }
+            //upper case
+            function upper_case(){
+                var str7 = document.getElementById("val8").value;
```

Comment:
do not add numbers in variable names instead use word for that like string_seven, varibale name should be identical